### ISSUE TITLE
[BUG] Rancher Harvester RKE1 provisioning is failing with k8s v1.24

### ISSUE THREAD

**Issue by noahgildersleeve** [harvester/harvester#2588]
**Describe the bug**

RKE1 and RKE2 
**To Reproduce**
1. Import Harvester into Rancher
2. Create cloud credentials
3. Create a node template
4. Create a RKE1 cluster

**Expected behavior**

The RKE cluster should provision
**Support bundle**
[supportbundle_e2184c5c-e3fa-4140-8d69-14729e17f224_2022-07-28T07-49-10Z.zip](https://github.com/harvester/harvester/files/9206848/supportbundle_e2184c5c-e3fa-4140-8d69-14729e17f224_2022-07-28T07-49-10Z.zip)
**Environment**

- Harvester ISO version: v1.0.3-rc1
- Rancher version: v2.6.7-rc5 single node Docker
- Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): Single node KVM virtualized node

**Additional context**
I was seeing this issue with both RKE2 and RKE1 at first, but after clearing out them and trying again I was only able to reliably reproduce the RKE1 issues with v1.24 k8s. It seems to be working on v1.23.
![Screenshot_20220728_005021](https://user-images.githubusercontent.com/83787952/181453538-ffce9c8a-06f0-476b-8b75-29de5f574a3a.png)
![Screenshot_20220728_010045](https://user-images.githubusercontent.com/83787952/181453593-f52265a7-4672-4f0e-85e9-647e42107aaf.png)
![Screenshot_20220728_005021](https://user-images.githubusercontent.com/83787952/181453597-8d9b20da-cf93-493e-9725-642b479ccffd.png)

**Comment by chrisho** [comment]
After testing and verification,  RKE v1.23 has no reproduction issues, but RKE2 v1.24-rk2 needs to reserve enough memory, at least 3G running Kube service, and the resource.request.memory is configured in the static pod file.
When you test RKE again, you can try to configure more memory for machines.

<img width="1525" alt="image" src="https://user-images.githubusercontent.com/3347460/181778269-1c8975bd-04bc-4c6e-990f-6d5964f31d69.png">
<img width="1576" alt="image" src="https://user-images.githubusercontent.com/3347460/181778379-a671e88a-5f9d-40e8-a9f5-1791694b81b6.png">

In the RKE node template or RKE2 node pool configuration UI, the default value for resource configuration is 2C 4G, k8s services have a minimum requirement for resources.
We can consider when the user is configured with too low resources:
1. Frontend validation adds minimum resource validation.
2. Add documentation or hint instructions.

If the retest passes, it can be moved to v1.1.0 as an enhanced issue solution

**Comment by noahgildersleeve** [comment]
Tested in v2.6.7-rc6 and v1.0.3-rc2. I updated the defaults for the template in RKE2 to 4 cores and 8gigs of RAM. It deployed successfully. We might want to update our defaults or add something to the FAQ about it or in the troubleshooting docs
![Screenshot_20220804_151744](https://user-images.githubusercontent.com/83787952/182962307-3a11eb25-2169-4b69-84f3-33ee6e192b5f.png)
![Screenshot_20220804_151735](https://user-images.githubusercontent.com/83787952/182962309-6a1990ca-8b3d-4b10-9290-eb423c30b6c5.png)
![Screenshot_20220804_134150](https://user-images.githubusercontent.com/83787952/182962310-876fc79e-dd53-446c-9441-392e91809206.png)
